src/Components/AddPlace.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import '../AddPlace.css';
+
+function AddPlace() {
+  const initialPlaceData = {
+    description: '',
+    photo: '',
+    location: '',
+    rate: '',
+    address: '',
+    pricepernight: '',
+    user_id: localStorage.getItem('userId'),
+  };
+  const [placeData, setPlaceData] = useState(initialPlaceData);
+  const [status, setStatus] = useState('idle');
+  const [error, setError] = useState(null);
+
+  const handleChange = (e) => {
+    setPlaceData({ ...placeData, [e.target.name]: e.target.value });
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    setStatus('loading');
+
+    try {
+      const userId = localStorage.getItem('userId');
+      await axios.post(`http://localhost:3000/api/v1/users/${userId}/places`, placeData);
```

Comment:
- It looks like this is an action. The actions are part of redux architecture, I kindly invite you to add all actions inside the redux folder, preferably with all actions together in place to add in the components

- Please, consider the same for other components with the same issue 😄